design/ceph/admission-controller.md

```diff
@@ -61,7 +61,7 @@ openssl base64 -d -A -out ${PUBLIC_KEY_NAME}.pem
 
 **Sample : Certificate Signing Request (CSR) in Kubernetes**
 ```
-apiVersion: certificates.k8s.io/v1beta1
+apiVersion: certificates.k8s.io/v1
```

Comment:
`certificates.k8s.io/v1` was only available since K8s 1.19, right? So it seems we need the docs and CI to still specify v1beta1 for older versions.

Reply:
In CI for admission controller, we use cert-manager that gives certificate and all and this is an older design doc so I didn't change it when I was working on the admission controller. But, for other changes, we need to handle both.

Reply:
oh right, this is just a design doc, no need to handle both here